packages/@vue/cli/bin/vue.js

```diff
@@ -155,6 +155,27 @@ program
     loadCommand('upgrade', '@vue/cli-upgrade')(semverLevel, cleanArgs(cmd))
   })
 
+program
+  .command('info')
+  .description('print debugging information about your environment')
+  .action((cmd) => {
+    console.log(chalk.bold('\nEnvironment Info:'))
+    require('envinfo').run(
+      {
+        System: ['OS', 'CPU'],
+        Binaries: ['Node', 'Yarn', 'npm'],
+        Browsers: ['Chrome', 'Edge', 'Firefox', 'Safari'],
+        npmPackages: '/**/{*vue*,@vue/*/}',
+        npmGlobalPackages: ['vue-cli']
```

Comment:
In Vue CLI 3 it's `@vue/cli` rather than `vue-cli`

Reply:
Good to know. Will you be deprecating this quickly or should I include a check for both?

Reply:
`vue-cli` is now a sub dependency of `@vue/cli-init` so I think checking for `@vue/cli` only would be sufficient.

Reply:
Will change 👍